packages/cra-template/package.json

```diff
@@ -2,10 +2,12 @@
   "name": "@coveo/cra-template",
   "version": "1.24.1",
   "scripts": {
+    "snyk:prepare": "cp ../../package-lock.json ./package-lock.json && npm i",
     "lint": "prettier --check . && eslint .",
-    "npm:bump:template": "npm --no-git-tag-version version",
+    "npm:bump:template": "npm --no-git-tag-version version 0.0.0",
```

Comment:
I'm quickly reading the doc on npm version https://docs.npmjs.com/cli/v8/commands/npm-version and I'm not sure I understand what the `0.0.0` does here.

Can you explain ? From the script name it's looking like it's trying to "template" the version from something, but I'm not able to connect the dots.